### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Create Calculator.js file.
Add a basic design to the calculator.
Use class-based components.
Import Calculator.js file and use it in the main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
```
Comment (omar-labana) [inline src/App.js]: - [ ] The `App.js` component should be class-based.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <div className="calc-grid">
+          <button type="button"> AC </button>
+          <button type="button"> +/- </button>
+          <button type="button"> % </button>
+          <button type="button" className="logic-operators"> ÷ </button>
+          <button type="button"> 7 </button>
+          <button type="button"> 8 </button>
+          <button type="button"> 9 </button>
+          <button type="button" className="logic-operators"> x </button>
+          <button type="button"> 4 </button>
```
Comment (omar-labana) [inline src/components/Calculator.js]: - [ ] The `calculator output` is a bit broken, please update your styling so it matches the wireframe.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Danieldotcomcoder, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @Danieldotcomcoder   🙋‍♀️, 

# STATUS APPROVED ✔

Thanks for making the requested changes 👏 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**